barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/ultra_recursive_verifier.cpp

```diff
@@ -128,16 +128,14 @@ std::array<typename Flavor::GroupElement, 2> UltraRecursiveVerifier_<Flavor>::ve
     // multivariate evaluations at u
     const size_t log_circuit_size = numeric::get_msb(static_cast<uint32_t>(key->circuit_size));
     auto sumcheck = Sumcheck(log_circuit_size, transcript);
+
     RelationSeparator alpha;
     for (size_t idx = 0; idx < alpha.size(); idx++) {
         alpha[idx] = transcript->template get_challenge<FF>("alpha_" + std::to_string(idx));
     }
 
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/1041): Once hashing produces constraints for Ultra in
```

Comment:
put this in the PR description

Reply:
I'm not opposed to sprinkling TODOs in the code FWIW it helps keep them top of mind

Reply:
this TODO is supposed to be resolved by this PR